src/onelogin/saml2/authn_request.py

```diff
@@ -77,9 +83,11 @@ def __init__(self, settings, force_authn=False, is_passive=False):
                 authn_comparison = security['requestedAuthnContextComparison']
 
             if security['requestedAuthnContext'] is True:
-                requested_authn_context_str = """    <samlp:RequestedAuthnContext Comparison="%s">
-        <saml:AuthnContextClassRef>urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport</saml:AuthnContextClassRef>
-    </samlp:RequestedAuthnContext>""" % authn_comparison
+                if 'activeDirectoryFederation' in security and security['activeDirectoryFederation']:
```

Comment:
Don't need that.

Instead set as security settings, the value 'requestedAuthnContext' as urn:federation:authentication:windows
as described in the [documentation](https://github.com/onelogin/python-saml#settings)